apps/etterlatte-testdata-behandler/src/main/kotlin/no/nav/etterlatte/testdata/automatisk/BeregningService.kt

```diff
@@ -0,0 +1,46 @@
+package no.nav.etterlatte.testdata.automatisk
+
+import com.github.michaelbull.result.mapBoth
+import no.nav.etterlatte.beregning.grunnlag.LagreBeregningsGrunnlag
+import no.nav.etterlatte.libs.common.beregning.BeregningsMetode
+import no.nav.etterlatte.libs.common.beregning.BeregningsMetodeBeregningsgrunnlag
+import no.nav.etterlatte.libs.common.retryOgPakkUt
+import no.nav.etterlatte.libs.ktor.ktor.ktorobo.DownstreamResourceClient
+import no.nav.etterlatte.libs.ktor.ktor.ktorobo.Resource
+import no.nav.etterlatte.libs.ktor.token.Systembruker
+import no.nav.etterlatte.testdata.BEGRUNNELSE
+import java.util.UUID
+
+class BeregningService(
+    private val klient: DownstreamResourceClient,
+    private val url: String,
+    private val clientId: String,
+) {
+    suspend fun beregn(behandlingId: UUID) =
+        retryOgPakkUt {
+            klient.post(Resource(clientId, "$url/api/beregning/$behandlingId"), Systembruker.testdata) {}.mapBoth(
+                success = {},
+                failure = { throw it },
+            )
+        }
+
+    suspend fun lagreBeregningsgrunnlag(behandlingId: UUID) =
+        retryOgPakkUt {
+            klient.post(
+                Resource(clientId, "$url/api/beregning/beregningsgrunnlag/$behandlingId/barnepensjon"),
```

Comment:
Bør denne ta høyde for ytelse?

Reply:
Jepp! Hadde eigentleg utsatt det, men det er jo trivielt å få på plass, så fiksar det no.

Og så kan ein jo spørre seg korfor vi har separate endepunkt for dette, for implementasjonane ser kliss like ut, men tar ikkje den ryddinga akkurat no.